surveys/2025/annual-survey/translations/ru.md

```diff
@@ -401,25 +401,25 @@ Type: select all that apply
 - Я совмещаю Cargo и другую систему сборки
 - Если вы используете Cargo с другими системами сборки (или просто используете их), какие из них вы используете?
 
-### Откуда вы загружаете зависимости (пакеты/crates) для сборки проектов на Rust?
+### Откуда вы загружаете зависимости (crates) для сборки проектов на Rust?
 
 Type: select all that apply
 
-- Я использую crates.io
-- Я использую собственный/локальный/корпоративный реестр
-- Я использую зеркало crates.io
+- crates.io
+- Собственный/локальный/корпоративный реестр
+- Зеркало crates.io
+- Git репозитории
 - Я не знаю
-- 
-- Other
+- Другое
 
-### 
+### Используете ли вы коды ошибок компиляции?
```

Comment:
@Kobzol, I can see that the translation exists for the description, but you latest commit doesn't contain it. That's weird.

Reply:
I think comment is also resolved, can you confirm @YohDeadfall ?